pkg/proxy/config/config.go

```diff
@@ -22,9 +22,11 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	discovery "k8s.io/api/discovery/v1"
+	networking "k8s.io/api/networking/v1alpha1"
```

Comment:
call this `networkingv1alpha1`; we might need to pull in `networkingv1` at some point too.

(yes, discovery is wrong)